core/server/data/migrations/versions/4.5/03-remove-unique-constraint-from-product-name.js

```diff
@@ -0,0 +1,16 @@
+const {logging} = require('../../../../../shared');
+const {createNonTransactionalMigration} = require('../../utils');
+
+module.exports = createNonTransactionalMigration(
+    async function up(knex) {
+        logging.info('Dropping UNIQUE constraint from products.name');
+        return knex.schema.alterTable('products', (table) => {
```

Comment:
@allouis we should be reusing our existing helpers where possible :wink: 

Could be something like:
```

const {dropUnique} = require('../../../schema/commands');

...

return await dropUnique('products', 'name', knex);
```

Reply:
Nice, thanks!